app/gateway-oss/2.4.x/hybrid-mode.md

```diff
@@ -290,6 +290,27 @@ cluster_cert_key = data-plane.crt
 `cluster_server_name` specifies the SNI (Server Name Indication extension) to use for data plane
 connections to the control plane through TLS. When not set, data plane will use `kong_clustering` as the SNI.
 
+## Revocation check of Data Plane certificates
+
+When Kong is running Hybrid mode with PKI support, the Control Plane can be configured to
+optionally check for revocation status of the conecting Data Plane certificate.
+
+The supported method is through Online Certificate Status Protocol (OCSP) respoders.
+In addition, Data Plane certificates issued must contain the "Certificate Authority Information Access" extension
+which points to the URI of which the OCSP responder may be reached from the Control Plane.
+
+To enable OCSP checks, set the `cluster_ocsp` config on the Control Plane to one of the following values:
+
+* `on`: OCSP revocation check is enabled and Data Plane must pass the revocation check in order to establish connection
+with Control Plane. This implies that certificates without the OCSP extension or unreachable OCSP responder also prevents
+connection from being established.
+* `off`: OCSP revocation check is disabled (this is the default).
+* `optional`: OCSP revocation check will be attempted, however, if the required extension is not found inside Data Plane
+provided certificate or communication with the OCSP responder failed, then Data Plane is still allowed through.
+
+Note that OCSP checks are only performed on Control Plane against certificates provided by incoming Data Plane
+nodes. The `cluster_ocsp` config has no effect on Data Plane nodes.
+
```

Comment:
As `cluster_oscp` affects telemetry as well and there's no `cluster_telemetry_ocsp`, and possibly generic messaing in the future, we could also mention:
```suggestion
`cluster_oscp` affects all Hybrid mode connections from Data Plane to Control Plane.

```